tests/integration/240.telemetry.test.js

```diff
@@ -49,9 +49,40 @@ test.serial('should track --telemetry-enable', async (t) => {
 test('should send netlify-cli/<version> user-agent', async (t) => {
   await withMockApi(routes, async ({ apiUrl, requests }) => {
     await callCli(['api', 'listSites'], getCLIOptions(apiUrl))
-    t.true(requests.length !== 0)
+    const request = requests.find(({ path }) => path === '/api/v1/track')
```

Comment:
This test was failing on my machine as the mock api can receive requests in a different order.
Updated the test to better search the analytics request